tests/testthat/_snaps/error-call.md

```diff
@@ -207,8 +207,8 @@
     Output
       <error/vctrs_error_subscript_oob>
       Error in `my_function()`:
-      ! Can't subset elements past the end.
-      i Location 10 doesn't exist.
+      ! Can't subset elements with `10`.
+      x Location must be less than or equal to 2, not 10.
```

Comment:
Could this be one line like `Can't find location 10; input is length 2`?

Reply:
This is slightly problematic because these errors are used for subsetting across tibble rows and cols. So it's hard to find generic formulations that work well for all cases.

Reply:
We could say "input has 2 elements" or "input has 3 rows". This is how we currently phrase it in an additional info bullet, but I guess you're saying this information should be closer?

Another difficult aspect is how to handle the case where multiple locations are not found without creating a too long line. To address this, I try to consistently mention multiple locations in their own bullet in this PR. But then it's hard to keep the length info close.